rules/color.js

```diff
@@ -9,6 +9,4 @@ module.exports = {
   'color-no-hex': true,
   // Disallow invalid hex colors.
   'color-no-invalid-hex': true,
-  // Disallow colors that are suspiciously close to being identical.
-  'no-indistinguishable-colors': true,
```

Comment:
There isn't a useful link.  The maintainers' general response to carping about deprecated rules is, "wanna write a plugin?"